src/components/MonthlyGood/AlreadySubscribedNotice.vue

```diff
@@ -1,7 +1,7 @@
 <template functional>
 	<div v-if="!props.onetime">
 		<h1 class="tw-text-center tw-text-h2 tw-mb-2">
-			You're already signed up for Monthly Good
+			You're already signed up for a subscription
```

Comment:
I changed the wording here to be able to reuse this messaging for users who have MG or Causes